src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,128 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import './TodoList.scss';
+import { Todo } from '../../type/todo';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+interface Props {
+  todos: Todo[],
+  selectedUserId: number,
+  handleSelectedUserId: (userId: number) => void,
+}
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
+const OShowTodos = {
+  All: 0,
+  Active: 1,
+  Completed: 2,
+} as const;
+
+type ShowTodos = typeof OShowTodos[keyof typeof OShowTodos];
```

Comment:
create enum for this case

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Done